packages/ui/Scrollable/Scrollable.js

```diff
@@ -589,6 +613,10 @@ class ScrollableBase extends Component {
 		this.isDragging = true;
 		this.dragStartX = this.scrollLeft + this.getRtlX(ev.x);
 		this.dragStartY = this.scrollTop + ev.y;
+
+		if (this.props.onDragStart) {
```

Comment:
These checks aren't required as `forward` will safely bail if the event doesn't exist on `props`.